SavedInstances/Core/Core.lua

```diff
@@ -1640,6 +1661,18 @@ hoverTooltip.ShowToonTooltip = function (cell, arg, ...)
   if t.RBGrating and t.RBGrating > 0 then
     indicatortip:AddLine(BG_RATING_ABBR, t.RBGrating)
   end
+  if t.SoloShuffleSpec1rating and t.SoloShuffleSpec1rating > 0 then
+    indicatortip:AddLine("Solo Shuffle Rating: " .. t.Spec1Name, t.SoloShuffleSpec1rating)
```

Comment:
Maybe combining `PVP_RATED_SOLO_SHUFFLE` and `RATING`. And the following.

Reply:
Good idea. Fixed!